ArjunaJTA/object_store/run-jgroups-cluster.sh

```
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Demonstrates two transaction manager nodes sharing a JGroups ReplCache store.
+# Node1 creates an in-doubt transaction, node2 joins the cluster and verifies
+# it can see node1's transaction (proving cross-node recovery is possible).
+
+set -m
+
+echo "JGroups Cluster Object Store Quickstart"
+echo "========================================"
+
+[ "x$QUICKSTART_NARAYANA_VERSION" != 'x' ] &&\
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)